### ISSUE TITLE
Moving profile to `about` and retaining a two-column theme

### ISSUE THREAD

**Issue by kleinbottle** [digitalcraftsman/hugo-icarus-theme#110]
I am learning how to tweak the `hugo-icarus-theme `and have met with small successes so far. Your partials are very good teaching tools!

I have disabled both the profile and widgets for the blog posts as suggested at the theme website, but the main text still occupies only about half the page and is on the left. I would have hoped that it would be centred and occupy perhaps three-quarters of the available width.

How can I get it to do this?

Also, I would appreciate pointers on what to do to move the profile to an "about" page. I think it would need a special single page template to do this. Is that correct?

Many thanks.

**Comment by mandarvaze** [comment]
@kleinbottle Can you share your changes ? I think it will be useful to lot of folks.

**Comment by kleinbottle** [comment]
@mandarvaze: I have not made any changes to the theme as such, but rather just resized the browser window and invoked these settings in the preamble to a test post:
+++
disable_profile = true
disable_widgets = true
+++

I found then that the width of the main text occupied only half the browser window width whereas I would have expected about two thirds or three quarters. This was the main sticking point.

When I tried to change this behaviour, I found certain hardcoded percentages in the CSS, probably for responsive behaviour. I could not change those values and retain robust behaviour. Also, I do not know enough about responsive design to do this correctly.

I have not put in thought or effort on this for some months. I now realize that my ideal theme would have two rather than three columns and I am looking at suitable examples on the Hugo site to do this.

**Comment by mandarvaze** [comment]
>  I now realize that my ideal theme would have two rather than three columns and I am looking at suitable examples on the Hugo site to do this.

I would also like two column variation of icarus.

Profile details on the left **all the time** is sorta waste of space. May be 2/3rd on the left for the main blog post and one third to the right for "Recent". Is that what you are considering ?

Do keep this issue updated if you make any progress.

**Comment by kleinbottle** [comment]
@mandarvaze: I agree with you entirely. I will post here when  find a useful two-column theme.
